foundations/javascript_basics/variables_and_operators.md

```diff
@@ -1,86 +1,128 @@
 ### Introduction
 
-In the previous sections you learnt how to structure webpages with HTML and style them with CSS. The next step is to make the webpage *interactive*, which is exactly what JavaScript is for. In this section, we will focus on the fundamentals of JavaScript and how you can use it to manipulate all the various interactions between the web page and user.
+In the previous sections you learned how to structure webpages with HTML and style them with CSS. The next step is to make the webpage *interactive*, which is exactly what JavaScript is for.
+
+In this section, we will focus on the fundamentals of JavaScript and how you can use it to manipulate all the various interactions between the web page and user.
 
 ### Lesson overview
 
 This section contains a general overview of topics that you will learn in this lesson.
 
-- How do you declare a variable?
-- What are three different ways to declare a variable?
-- Which one should you use when?
-- What are the rules for naming variables?
-- What are operators, operands, and operations?
-- What is concatenation and what happens when you add numbers and strings together?
-- What are the different types of operators in JavaScript?
-- What is the difference between == and ===?
-- What are operator precedence values?
-- What are the increment/decrement operators?
-- What is the difference between prefixing and postfixing them?
-- What are assignment operators?
-- What is the Unary Plus Operator?
+- Discuss the different ways you can declare variables.
+- Learn how to write mathematical expressions and the basic arithmetic operators.
+- What is operator precedence and some more advanced operators.
 
 ### How to run JavaScript code
 
-All JavaScript we will be writing in the majority of the Foundations course will be run via the browser. Later lessons in Foundations and the NodeJS path will show you how to run JavaScript outside of the browser environment. Outside of these lessons, for now you should always default to running your JavaScript in the browser unless otherwise specified, otherwise you may run into unexpected errors.
-
-The simplest way to get started is to create an HTML file with the JavaScript code inside of it.  Type the basic HTML skeleton into a file on your computer somewhere:
+The simplest way to get started is to create an HTML file with the JavaScript code inside of it. Use the VS Code snippet <kbd>!</kbd> + <kbd>TAB</kbd> to create the basic HTML skeleton in a file on your computer somewhere. Be sure to include the `<script>` tag:
```

Comment:
That paragraph about where JavaScript is run is critical. You can't just remove it.

Reply:
oops :eyes: 

restored it